core/src/main/java/org/mobilitydata/gtfsvalidator/notice/EmptyFileNotice.java

```diff
@@ -18,7 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** A CSV file is empty. */
+/**
+ * A CSV file is empty.
+ *
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
We do not enforce that check at Google, so this should be rather a warning. This is not a blocker for using feed data if, e.g., there is an empty file `pathways.txt`. An empty file is practically equivalent to a missing file. 

We have 3 situations that may be treated as equivalent:
* a file is missing
* a file has only headers
* a file is empty

GTFS marks certain files as required. I think that we should amend the GTFS reference and say that a required file must contain at least one record (not counting the headers).

Reply:
The Javadocs for `CsvFile.isEmpty()` currently say:

>Tells if the file is empty, i.e. it has no rows and even no headers.

And GTFS says:

>The first line of each file must contain field names. 

...so this seems to align as an ERROR to me. Let's discuss in a new issue/PR if needed so we can get this merged - it's getting hard to manage this PR due to length.